service/src/setup.rs

```diff
@@ -54,10 +55,10 @@ pub(crate) fn initialize_shard_and_keys(
 pub(crate) fn init_shard(enclave: &Enclave, shard_identifier: &ShardIdentifier) {
 	match enclave.init_shard(shard_identifier.encode()) {
 		Err(e) => {
-			println!("Failed to initialize shard {:?}: {:?}", shard_identifier, e);
+			println!("Failed to initialize shard {:?}: {:?}", shard_identifier.0.to_base58(), e);
```

Comment:
We should implement Display for the shard identifier, which is base58 encoded, see https://github.com/integritee-network/pallets/issues/213

Reply:
agreed. but doesn't need to be this PR